.github/workflows/pre-release.yml

```diff
@@ -31,12 +31,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ros_distro: [humble, foxy]
+        ros_distro: [humble]
```

Comment:
will add iron here only when it builds and be ready on the ROS2 public package index

Reply:
I say let's add it and ignore Iron failure, it should be fixed once it's merged , right?

Reply:
done